.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
```

Comment:
Separate workflow is definitely an overkill. IMHO it should be another step in Build workflow. You will also easily enable testing for all platforms.

Reply:
Agreed